### PR TITLE
78 connect exercise management with api

### PR DESCRIPTION
**Pull request by AnasAhakki**

### CODE REVIEW

File: src/components/admin/TaskManagement.vue [hunk 1]
```diff
@@ -222,7 +222,6 @@
 import {onBeforeMount, Ref, ref} from "vue";
 import {useI18n} from "vue-i18n";
 
-import MarkdownModal from "@/components/helpers/MarkdownModal.vue";
```
Comment (hwgn) [inline src/components/admin/TaskManagement.vue]: Why remove this?

Comment (AnasAhakki) [reply src/components/admin/TaskManagement.vue]: By Mistake !

**Review by hwgn — APPROVED** [review]
LGTM

**Review by hwgn — CHANGES_REQUESTED** [review]
Please pull from development!

**Review by hwgn — APPROVED** [review]

**Pull request merged**